doc/getting_started/installation_win.rst

```diff
@@ -3,10 +3,6 @@
 Install Windows Host Dependencies
```

Comment:
As for macOS, this is more a repetition plus troubleshooting. Maybe turn it into a pure troubleshooting section?